app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @review = Review.new(params_review)
+    @review.user_id = current_user.id
+    @review.video_id = params[:video_id]
+    @review.rating = params[:Rating]
```

Comment:
If you look into params just after reviews#create is called, there is actually no ```rating``` attribute in the ```review``` hash. This is because, although I did use ```form_for [@video, @review]``` on my ```videos#show``` template, I couldn't figure out how to add ```select``` tags using the modeled back form, so I used  ```select_tag``` as so:

```
= form_for [@video,@review] do |f|
        = f.label :title
        = f.text_field :title
        = f.label :rating
        = select_tag("Rating", options_for_select([['One Star',1],['Two Stars',2],['Three Stars',3],['Four Stars',4],['Five Stars',5]]))
        = f.label 'Review'
        = f.text_area :body
        = f.submit 'Post Review'
```

So when ```reviews#create``` is called, I had to manually set the ```review.rating``` attribute. On the ```videos#show``` (where the form is), ```rating``` is not the same as ```Rating```. 

```
  @review.rating = params[:Rating] # :Rating is its own hash, separate from the review hash
```

Now that I am seeing this again, there is no need for ```:rating``` in my ```params_review()``` so I'll remove it. So in summary,  I wrote it this way because I couldn't figure out how to do it with ```form_for```.